Bikeini/screens/BikeInformation.js

```diff
@@ -417,7 +450,7 @@ class BikeInformation extends React.Component {
     const imgSource = bikeData.image_url ? { uri: bikeData.image_url } : stockBicycle;
 
     return (
-      <View style={styles.container}>
+      <KeyboardAvoidingView style={styles.container} behavior="padding" enabled>
```

Comment:
Im using the same component for SignUp.js and it works great on both my emulator and real device! 😸